metrics/default_metrics/tarantool/operations.lua

```diff
@@ -0,0 +1,18 @@
+local utils = require('metrics.default_metrics.tarantool.utils');
+
+local function get_operations_metrics()
+    if not utils.box_is_configured() then
+        return
+    end
+
+    local current_stat = box.stat()
+
+    for k, v in pairs(current_stat) do
+        utils.set_gauge('stats_op_' .. k:lower() .. '_total', 'Total amount of ' .. k:lower() .. 's', v.total)
+        utils.set_gauge('stats_op_' .. k:lower() .. '_rps', 'Total RPS ' .. k:lower() .. 's', v.rps)
+    end
+end
+
+return {
+    init = get_operations_metrics
```

Comment:
Maybe `enable` instead of `init`?

We usually use `init` to configure a module for later usage. Think of it as a constructor.

This module doesn't have any state to configure.